### PR TITLE
[SlackBlockBuilder] fix _parse_url_blocks

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [link to the issue
](https://jira-dc.paloaltonetworks.com/browse/XSUP-34838)
## Description
Fixed an issue where the blocks were not parsed correctly.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.py [hunk 1]
```diff
@@ -157,10 +157,18 @@ def _parse_url_blocks(self):
         Slack provides a tool located at https://app.slack.com/block-kit-builder. When you are done, you can copy and
         paste the URL into the blocks_url argument for this automation. The URL is then decoded to provide the blocks.
         """
-        url_encoded_blocks: str = self.url.split('#')[1]
-        url_decoded_blocks: str = urllib.parse.unquote(url_encoded_blocks)
-        parsed_blocks: Any = json.loads(url_decoded_blocks)
-        self.blocks_dict = parsed_blocks.get('blocks', [{}])
+        try:
+            if match_url_encoded_blocks := re.search(r"#(.+)", self.url):
```
Comment (bziser) [inline Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.py:161]: maybe add in the description of this function why the regex needed and been used

File: Packs/Slack/ReleaseNotes/3_4_9.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### SlackBlockBuilder
+
+- Fixed an issue where the blocks were not parsed correctly.
```
Comment (bziser) [inline Packs/Slack/ReleaseNotes/3_4_9.md:6]: please elaborate in which case

**Review by bziser — APPROVED** [review]

File: Packs/Slack/ReleaseNotes/3_4_9.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### SlackBlockBuilder
+
+- Fixed an issue where the blocks were not parsed correctly, when the *blocks_url* contains more than one `#` symbol.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/ReleaseNotes/3_4_9.md:6]:
```suggestion
- Fixed an issue where the blocks were not parsed correctly, when the *blocks_url* parameter contains more than one `#` symbol.
```

**Comment by ShirleyDenkberg** [comment]
@bziser @rshunim @amshamah419 Doc review completed.

**Pull request merged**